apps/ui/src/types/community-type.ts

```diff
@@ -0,0 +1,8 @@
+import { Community, UserStatus } from '@communecar/types';
+
+export type ClientCommunity = Community & {
```

Comment:
I think the name is a bit off, what is this `ClientCommunity`? Isn't it just a regular community?